frontend/libs/health-department/feature-case-detail/src/lib/case-detail/case-detail.component.ts

```diff
@@ -254,18 +116,14 @@ export class CaseDetailComponent implements OnDestroy {
   closeCase(halResponse: HalResponse) {
     this.subs.sink = this.apiService
       .deleteApiCall<any>(halResponse, 'conclude')
-      .pipe(switchMap(() => this.healthDepartmentService.getCase(this.caseId)))
+      .pipe(switchMap(() => this.entityService.getByKey(this.caseId)))
       .subscribe((data) => {
         this.snackbarService.success('Fall abgeschlossen.');
-        this.updatedDetail$$.next(data);
+        this.entityService.updateOneInCache(data);
```

Comment:
ist das nicht doppelgemoppelt? getByKey sollte doch den Cache bereits aktualisieren, oder?

Reply:
Aus meiner Sicht sollte hier in die Übersicht zurücknavigiert werden. Der Fall ist abgesclhossen und sollte damit nicht mehr bearbeitbar sein. Außerdem könnte man den Fall mehrmals abschließen, wenn man in dieser Ansicht bleibt